### ISSUE TITLE
Make null values inherently htmlSafe

### ISSUE THREAD

**Issue by Frozenfire92** [emberjs/ember.js#12681]
If I don't have a property set on a controller yet (could be set on an init function or similar) then you will get the deprecation notice for binding style attributes

### Example
```js
someInit: function(){
  this.set('myProp', Ember.String.htmlSafe('width: 50%;');
}
```
```hbs
<div style={{myProp}}>...</div>
```
To fix this you must explicitly do the following (which seems a little overkill)
```js
myProp: Ember.String.htmlSafe(""),
someInit: function(){
  this.set('myProp', Ember.String.htmlSafe('width: 50%;');
}
```

**Comment by robbiespeed** [comment]
+1

**Comment by pixelhandler** [comment]
@rwjblue can this be closed? since #12686 prevents `undefined` and `null` from generating an htmlsafe warning

**Comment by rwjblue** [comment]
Yep, thanks @pixelhandler!
